hummingbot/connector/derivative/kucoin_perpetual/kucoin_perpetual_constants.py

```diff
@@ -89,6 +89,11 @@
 WS_CONNECTION_TIME_INTERVAL = 20
 WS_REQUEST_LIMIT_ID = "WSRequest"
 
+ORDER_CHANGE_EVENT_TYPE = "orderChange"
+BALANCE_EVENT_TYPE = "availableBalance.change"
+POSITION_CHANGE_EVENT_TYPE = "position.change"
+TRADE_EVENT_TYPE = "match"

```

Comment:
this should be added to the dictionary of OrderState

Reply:
I'm a bit confused as to why these would be added to the OrderState dictionary. These aren't order states, they are the types of WS events being returned. This is a separate piece of data from the state of the order, as I understand it. Apologies if I'm off base, just want to be sure this is done right.